### ISSUE TITLE
hover-triggered popovers disappear after opening, or teleport to top-left of container

### ISSUE THREAD

**Issue by RentecTravis** [twbs/bootstrap#33340]
Using a value of `hover` for the `trigger` option with popovers causes strange behavior. Sometimes the popover opens in the correct position, and then disappears immediately. Other times the popover teleports to the top left of the viewport, but only after moving the mouse cursor.

[View Codepen example](https://codepen.io/tvanc/pen/KKNbJve)

_May_ be duplicate of #32897 and #32372.

32897 just says "not working", and is for beta1. 32372 is specifically about tooltips, not popovers.

OS: Windows 10
Browser: Chrome, Version 89.0.4389.82 (64-bit)

![bs5-hover-popover-issue](https://user-images.githubusercontent.com/42187666/110853673-c1872000-8268-11eb-821c-e10fbe9fbb25.gif)

Edit: fixed link to codepen example

**Comment by alpadev** [comment]
Popovers are based on the Tooltip component so this should be fixed with https://github.com/twbs/bootstrap/pull/33289.

**Comment by mdo** [comment]
Closing since https://github.com/twbs/bootstrap/pull/33289 was merged.
